client/CPlayerInterface.cpp

```diff
@@ -322,6 +322,9 @@ void CPlayerInterface::acceptTurn()
 		else
 			logGlobal->warn("Player has no towns, but daysWithoutCastle is not set");
 	}
+	
+	JsonNode reply(JsonNode::JsonType::DATA_NULL);
+	cb->sendQueryReply(reply, queryID);

```

Comment:
Perhaps use selectionMade call? And not this weird version that sends Json for some reason?
```suggestion
	cb->selectionMade(0, queryID);
```